core/templates/services/guppy-initialization.service.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for initializing guppy instances.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
```

Comment:
the import order in general is:

1. Core angular imports
2. Third-party imports
3. Application imports

Within each sub-category you should sort by the module it is imported from and not what is being imported.

So something like:
```
import { A } from 'D';
import { C } from 'B';
```
is wrong.

The right way to do that would be:
```
import { C } from 'B';
import { A } from 'D';
```
because 'B' < 'D'

Please change the imports to follow this order.

Reply:
Thanks for such a clear explanation! I've reordered the imports.

Reply:
Done!